client/i18n/locales/english/translations.json

```diff
@@ -629,8 +630,8 @@
     "progress-modal-cta-10": "Donate now to help us develop free professional programming certifications for all.",
     "help-us-reach-20k": "Donate now to help our charity reach our goal of 20,000 monthly supporters this year.",
     "beta-certification": "This certification is currently in beta. Please consider donating to support the completion of its development.",
-    "unfinished-certification": "This certification is currently in active development. While there isn't a claimable certification available at the moment, one will be available soon. In the meantime, you're welcome to explore the courses we have created below.",
-    "consider-donating": "Please consider donating to support the completion of its development.",
+    "unfinished-certification": "This certification will take you a substantial amount of time and effort to complete. If you start now, you may be ready to take the final exam when we launch it in the coming months.",
```

Comment:
Do we want this copy as a separate key? Or are we confident we won't ship any other beta content until this is fully done?

Reply:
I think this will be what we want if we do ship anything else. This will be shown on the two English superblock pages as well. We decided that was fine.

Reply:
Decided to just make new keys. Maybe we'll want to revert or use them somewhere else or something.